README.md

```diff
@@ -196,6 +196,14 @@ yalc add @dasch-swiss/dsp-js
 npm install
 ```
 
+### Run the application locally in dev mode with Live Reloading
+
+To make changes to the DSP-UI and have your changes reflected immediately upon saving, follow these steps:
+
+ - run `npm run build-watchful-lib-dev` from the project root directory
+ - open a new tab and `run npm run start` from the project root directory
```

Comment:
the first run should be outside of the quote

Reply:
🤦 fixed in 7076dea